### ISSUE TITLE
Odd output behavior for decimal values queried from yaml three dots elipsis 

### ISSUE THREAD

**Issue by chrisinmtown** [openuado/niet#57]
I noticed strange output behavior in version 1.8.0 from a yaml file with a valid decimal value (an unquoted number with a decimal point). When specifying no output format for niet, I was baffled to see niet output the value, then a newline character, then three periods (elipsis), then another newline character!

Here's the YAML file:

```
---
one: 1
two: 2.000000002
three: 3.3.3
```

Here's the behavior:

```
(venv-py37) % niet .one numbers.yaml 
1
(venv-py37) % niet .two numbers.yaml 
2.000000002
...
(venv-py37) % niet .three numbers.yaml
3.3.3
(venv-py37) % niet --version
niet version 1.8.0
```

I'm using "-f newline" as a workaround. Is there a better option?  

```
(venv-py37) % niet -f newline .two numbers.yaml               
2.000000002
```

**Comment by 4383** [comment]
Hello,

Thank you to reported this issue.

This is look like to a bug I will take a look soon in debug mode to see what happen here.

**Comment by chrisinmtown** [comment]
Thanks for confirming, I'll update the title to drop "Document blah blah"

**Comment by 4383** [comment]
@chrisinmtown thank you :)

**Comment by 4383** [comment]
In addition I did some tests with PyYAML and I obtain the following output:
```shell
$ python3.7 -c 'import yaml; print(yaml.dump(1,  default_flow_style=False))'      
1
...

$ python3.7 -c 'import yaml; print(yaml.dump(2.0000002,  default_flow_style=False))'
2.0000002
...

$ python3.7 -c 'import yaml; print(yaml.dump("3.3.3", default_flow_style=False))'
3.3.3
...

```

In all the cases all the output have the dots even if `niet` do the same things under the hood... I guess the `niet` treatment introduce this change, I need to dig more.

**Comment by 4383** [comment]
Gotcha! 

the error come from:
https://github.com/openuado/niet/blob/master/niet/__init__.py#L132

We check for int, str, list, etc... so `3.3.3` and `1` pass there but `2.0000002` is a float so it is ignored there and it use the `yaml` output format rather than the `newline` format normally used for single values, so that explain why we see difference between the `PyYAML` direct usage and differences between the different values.

**Comment by 4383** [comment]
@chrisinmtown  I proposed a fix (#62), feel free to review it.

Thanks
